Assets/SuperCharacterController/SuperCharacterController/Core/SuperCharacterController.cs

```diff
@@ -648,8 +648,13 @@ public void ProbeGround(Vector3 origin, int iter)
                 // it is connected to at it's base, if there exists any
                 if (Vector3.Angle(nearHit.normal, up) > superColType.StandAngle || nearHit.distance > Tolerance)
                 {
-                    var col = nearHit.collider.gameObject.GetComponent<SuperCollisionType>();
-
+                    SuperCollisionType col = null;
+                
+                    if (nearhit.collider != null)
```

Comment:
You're totally right. I was convinced I commented after clicking on the Changes tab on GitHub (which shows the final state) but I clearly did not, not sure why. Anyway, sorry :)